CSVSource/README.md

```diff
@@ -45,6 +45,8 @@ name for each class is the class's fully qualified class name, *e.g.* "io.vantiq
 
 ## Server Config File
 
+(Please read to the [SDK's server config documentation](../extjsdk/README.md#serverConfig) first.)
+
 The server config file is written as `property=value`, with each property on its
```

Comment:
Seems like we could move all of this stuff (down thru `Setting up Vantiq`) to the extsdk readme, and then just refer folks there for the whole thing.  The less repetition, the better.